internal/namespace/namespacehandler/handler.go

```diff
@@ -64,7 +64,7 @@ func (h *handler) ListNamespaces(ctx context.Context, _ *rts.ListNamespacesReque
 	return &rts.ListNamespacesResponse{Namespaces: apiNamespaces}, nil
 }
 
-// swagger:route GET /namespaces namespaces getNamespaces
+// swagger:route GET /namespaces relationship getNamespaces
```

Comment:
`getNamespaces` is kinda generic and could clash in the future with another system that has a concept of namespaces. How about `getRelationshipNamespaces` or something similar?

Reply:
fixed.